package.json

```diff
@@ -20,15 +20,15 @@
     "deepmerge": "4.2.2",
     "emoji-regex": "8.0.0",
     "form-data": "3.0.0",
-    "fuse.js": "3.6.1",
+    "fuse.js": "5.0.10-beta",
     "intl": "1.2.5",
     "jail-monkey": "2.3.2",
     "mime-db": "1.43.0",
     "moment-timezone": "0.5.26",
     "prop-types": "15.7.2",
     "react": "16.13.0",
     "react-intl": "2.8.0",
-    "react-native": "0.61.5",
+    "react-native": "0.62.0",
```

Comment:
Should we move to `0.62.1` directly?

This release also has @emilioicai 's PR https://github.com/facebook/react-native/commit/7a4753d76aab1c52a09f26ec6f7fd43a68da8a97

Reply:
I can do it in the deps PR as there is still no v8@0.62.1 yet